docs/configuration/configuration.md

```diff
@@ -444,6 +448,55 @@ tls_config:
 [ refresh_interval: <duration> | default = 60s ]
 ```
 
+### `<dockerswarm_sd_config>`
+
+Docker Swarm SD configurations allow retrieving scrape targets from [Docker Swarm](https://docs.docker.com/engine/swarm/)
+engine.
+
+The following meta labels are available on targets during [relabeling](#relabel_config):
+
+
+* `__meta_dockerswarm_endpoint_port_name`: the name of the endpoint port, if available
+* `__meta_dockerswarm_endpoint_port_publish_mode`: the publish mode of the endpoint port
+* `__meta_dockerswarm_service_id`: the id of the service
+* `__meta_dockerswarm_service_label_`: each label of the service
+* `__meta_dockerswarm_service_name`: the name of the service
+* `__meta_dockerswarm_updating_status`: the status of the service, if available
+* `__meta_dockerswarm_virtual_ip_address`: the address of the virtual ip
+* `__meta_dockerswarm_virtual_ip_network_id`: the network id of the virtual ip
+
+```yaml
```

Comment:
Host is missing

Reply:
That information is not exposed by the API

Reply:
There migth be a solution to this

Reply:
I mean you haven't documented a config field.

Reply:
oops ;)